docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -370,14 +597,43 @@ In this scenario, you'll add handling for sending a message. Your add-in will ch
-    > If your add-in wasn't automatically sideloaded, then follow the instructions in [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md#sideload-manually) to manually sideload the add-in in Outlook.
+    > If your add-in wasn't automatically sideloaded, follow the instructions in [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md#sideload-manually) to manually sideload the add-in in Outlook.
+
+1. In your preferred Outlook client, create a new message and set the subject. In the body, add some text. For example, "Here's a picture of the proposed logo."
+1. Send the message. A dialog appears requesting you to add an attachment. Select **Don't Send** or **Add an attachment**. The option available to you depends on whether you implemented the optional step to customize the **Don't Send** button.
+
+    ![Dialog requesting the user to add an attachment to the message.](../images/outlook-win-smart-alert.png)
+
+    ![Dialog with a customized Don't Send button requesting the user to add an attachment to the message.](../images/outlook-smart-alerts-custom-button.png)
+
+    > [!TIP]
+    > If you assign a task pane to the **Don't Send** button, closing the dialog also opens the specified task pane.
+
+1. Add an attachment to your message. If you implemented the optional step to customize the **Don't Send** button, use the task pane to add an attachment.
+
+    ![Task pane with an option to add an attachment.](../images/outlook-smart-alerts-task-pane.png)
+
+1. Send the message. There should be no alert this time.
+
+### Override the send mode option at runtime (optional)
+
+If you implemented the optional step to override the send mode option at runtime, perform the following to try it out.
+
+1. Run `npm start` in the root directory of your project. This starts the local web server (if it isn't already running) and sideloads your add-in.
+
+    > [!NOTE]
+    > If your add-in wasn't automatically sideloaded, follow the instructions in [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md#sideload-manually) to manually sideload the add-in in Outlook.
 
-1. In your preferred Outlook client, create a new message and set the subject. In the body, add some text. For example, "Hey, here's a picture of my dog!".
-1. Send the message. A dialog should pop up with a recommendation for you to add an attachment.
+1. In Outlook on Windows, create a new message and set the subject. In the body, add some text. For example, "Here's a picture of the proposed logo."
+1. From the ribbon, select **Contoso Add-in** > **Show Taskpane**.
+1. In the task pane, select **Add an inline image**. An image is added to the body of your message.
+1. Send the message. A dialog appears recommending to attach a copy of the image to the message.
+1. Select **Send Anyway** to send the message as-is, or select **Attach a copy** to include a copy before sending the message.
```

Comment:
See "as is" definition in [ahd](https://www.ahdictionary.com/word/search.html?q=as+is)

```suggestion
1. Select **Send Anyway** to send the message as is, or select **Attach a copy** to include a copy before sending the message.
```